src/reflection.jl

```diff
@@ -87,6 +87,20 @@ function find_callsites(CI::Core.CodeInfo, mi::Core.MethodInstance, slottypes; p
                 mi = get_mi(callsite)
                 if nameof(mi.def.module) === :CUDAnative && mi.def.name === :cufunction
                     callsite = transform(Val(:CuFunction), callsite, c, CI, mi, slottypes; params=params, kwargs...)
+                elseif callsite.info isa MICallInfo
+                    argtypes_ssa = map(c.args[3:end]) do a
+                        if isa(a, Core.SSAValue)
```

Comment:
I think you are missing Slot/Argument

Reply:
Thanks for catching it! I added the support for Core.Argument and Core.Slot:

https://github.com/JuliaDebug/Cthulhu.jl/blob/610abb4aa85822dde850851058314873509239e9/src/reflection.jl#L32-L39

https://github.com/JuliaDebug/Cthulhu.jl/blob/610abb4aa85822dde850851058314873509239e9/src/reflection.jl#L105-L108

Reply:
Apparently, we cannot use `CI.slottypes` because of Julia < 1.2? It's now:

https://github.com/JuliaDebug/Cthulhu.jl/blob/e2ec4434b01115cf265505b11b1490b930de979d/src/reflection.jl#L105-L106